core/src/main/scala/better/files/File.scala

```diff
@@ -968,14 +968,14 @@ class File private (val path: Path)(implicit val fileSystem: FileSystem = path.g
       Files.walkFileTree(
         path,
         new SimpleFileVisitor[Path] {
-          def newPath(subPath: Path): Path = destination.path.resolve(path.relativize(subPath))
+          def newPath(subPath: Path | Null): Path = destination.path.resolve(path.relativize(subPath))
 
-          override def preVisitDirectory(dir: Path, attrs: BasicFileAttributes) = {
+          override def preVisitDirectory(dir: Path | Null, attrs: BasicFileAttributes | Null) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.